diff --git a/arch/arm/boot/dts/artech.dts b/arch/arm/boot/dts/artech.dts
new file mode 100644
index 000000000..4730c9021
--- /dev/null
+++ b/arch/arm/boot/dts/artech.dts
@@ -0,0 +1,32 @@
+/*
+ * artech dts for raspberry pi 3 b model
+ */
+
+/ {
+	foo {
+		compatible = "artech,foo";
+	};
+
+	node@0 {
+		a-string-property = "A String";
+		a-string-list-property = "first string", "second stting";
+		a-byte-data-property = [01 23 34 56];
+
+		child-node@0 {
+			first-child-property;
+			second-child-property = <1>;
+			a-reference-to-something = <&node1>;
+		};
+
+		child-node@1 {
+		};
+	};
+
+	node1: node@1 {
+		an-empty-property;
+		a-cell-property = <1 2 3 4>;
+
+		child-node@0 {
+		};
+	};
+};
diff --git a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
index d9d9505c6..598c0104f 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
@@ -3,6 +3,7 @@
 #include "bcm2710.dtsi"
 #include "bcm283x-rpi-smsc9514.dtsi"
 #include "bcm283x-rpi-csi1-2lane.dtsi"
+#include "artech.dts"
 
 / {
 	compatible = "raspberrypi,3-model-b", "brcm,bcm2837";
